xill-ide/src/main/java/nl/xillio/xill/versioncontrol/JGitRepository.java

```diff
@@ -216,7 +216,7 @@ public void createBranch(String name) throws GitException {
             changes.addAll(status.getChanged().stream().map(s -> "*    " + s).collect(Collectors.toList()));
             changes.addAll(status.getConflicting().stream().map(s -> "#    " + s).collect(Collectors.toList()));
         } catch (GitAPIException | NoWorkTreeException e) {
-            LOGGER.error("Error retrieving changed files.", e);
+            //LOGGER.error("Error retrieving changed files.", e);
```

Comment:
Commented out code, can this be un-commented?

Reply:
It is/was expected to catch/fail according to the implemented test. It said "this should fail" and so it did. However it prevented the (successful) build/deploy.

Reply:
A `LOGGER.error()` statement, however, has no effect on the output of a function or side-effects; it only prints some characters to the 'console'. A test is never affected by these log statements. So I think there was something else wrong when you had failing tests. 

The thing now is, something went wrong (a `GitAPIException` or `NoWorkTreeException` occurred), and nothing is thrown or logged, so we can never know what went wrong now. Sonar will eventually complain, as we ignore exceptions by not logging them.